### ISSUE TITLE
Stable URL for all branches pointing to latest deployment of related branch

### ISSUE THREAD

**Issue by olmohake** [denoland/deploy_feedback#384]
### What problem are you trying to solve?

I am trying to setup a test environment. While this should be a quite simple task for the production environment, it is currently not that simple for environments related to branches other than main as the url for the deployed service changes with every deployment. 

### Describe the solution you'd like

It would be extremely helpful to have a stable url for all branches and let these urls automatically point to the latest deployment of each branch:

```<branchname>-<repository-name>.deno.dev```

### Describe alternatives you've considered

manual update of the url after each deployment for all branches/enviroments != main/production

**Comment by lucacasonato** [comment]
`$projectname--$branch.deno.dev` are stable branch URLs. The Deno Deploy GitHub bot adds this stable URL as a link to your PR in the "Checks" section.

![image](https://user-images.githubusercontent.com/7829205/236885744-0299d656-b8f9-49f6-a22e-b28404a35ede.png)

**Comment by olmohake** [comment]
Hmm, these don't show up anywhere for me. Is this only the case if i select the GitHub Action Integration or should this also be the case for the Automatic Integration option?

Also, if these url's are generated it would helpful to display theme on dash.deno.com!

**Comment by lucacasonato** [comment]
Yes, it is supported for both Automatic and GHA mode. You may have to accept updated permissions at https://github.com/settings/installations.

> Also, if these url's are generated it would helpful to display theme on dash.deno.com!

Noted!

**Comment by jvhellemond** [comment]
Is there any way to prevent branches from triggering a deploy? For example: GitHub's dependabot autmatically creates feature branches in my repository to bump dependencies. These should really not trigger a (branch)deploy. Can I somehow limit deployments to the main branch and maybe a few, specified branches?
